includes/CLI/Plugin_Check_Command.php

```diff
@@ -676,4 +700,33 @@ function ( $item ) use ( $warning_severity ) {
 
 		return array_merge( $errors, $warnings );
 	}
+
+	/**
+	 * Returns separated results by ERROR and WARNING.
+	 *
+	 * @since 1.1.0
```

Comment:
```suggestion
	 * @since 1.2.0
```

Should be upcoming version.